spring-boot/src/main/java/org/springframework/boot/logging/logback/DefaultLogbackConfiguration.java

```diff
@@ -126,26 +130,28 @@ private void base(LogbackConfigurator config) {
 
 	private Appender<ILoggingEvent> fileAppender(LogbackConfigurator config,
 			String logFile) {
-		RollingFileAppender<ILoggingEvent> appender = new RollingFileAppender<ILoggingEvent>();
+		RollingFileAppender<ILoggingEvent> appender =
+				new RollingFileAppender<ILoggingEvent>();
 		PatternLayoutEncoder encoder = new PatternLayoutEncoder();
-		String logPattern = this.patterns.getProperty("file", FILE_LOG_PATTERN);
+		String logPattern = this.resolver.getProperty("pattern.file", FILE_LOG_PATTERN);
 		encoder.setPattern(OptionHelper.substVars(logPattern, config.getContext()));
 		appender.setEncoder(encoder);
 		config.start(encoder);
 
 		appender.setFile(logFile);
 
-		FixedWindowRollingPolicy rollingPolicy = new FixedWindowRollingPolicy();
-		rollingPolicy.setFileNamePattern(logFile + ".%i");
+		SizeAndTimeBasedRollingPolicy<ILoggingEvent> rollingPolicy =
+				new SizeAndTimeBasedRollingPolicy<ILoggingEvent>();
+		rollingPolicy.setFileNamePattern(logFile + ".%d{yyyy-MM-dd}.%i.gz");
```

Comment:
It would be nice to have ability to specify fileName pattern by property `logging.file`.